### ISSUE TITLE
How about using Cloudflare Firewall Rules to reduce the number of scrapers hitting your server?

### ISSUE THREAD

**Issue by larsestrem** [JonasCz/How-To-Prevent-Scraping#9]
Great article!

Do you use a CDN firewall like Cloudflare?  I find their firewall rules useful.  

I would be interested to learn your take on using them.

**Comment by Kaos-Industries** [comment]
Agreed that Cloudflare and similar DDOS protection services is a significant omission from the article, I for one protect all my sites with Cloudflare's free protection and would be interested in knowing whether this contributes anything to protecting from scrapers. For example, a significant portion of the article is about detecting bot-like behaviour, but I've always been under the impression that this is something they automatically do as part of DDOS protection?

**Comment by garethjax** [comment]
you can filter traffic by ASN number and defeat most scrapers.
